### PR TITLE
refactor: core-ui and editor separate package/dependencies removed

### PR DESCRIPTION
**Pull request by abrehmaaan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,7 +58,17 @@
     "styled-components": "^5.2.1",
     "styled-system": "^5.1.5",
     "swr": "^1.3.0",
-    "types": "^0.1.1"
+    "types": "^0.1.1",
+    "@types/styled-system": "^5.1.10",
+    "@types/escape-html": "^1.0.0",
```
Comment (Fahad-Mahmood) [inline package.json]: these should be in dev dependencies

File: tslint.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "rules": { "no-empty-interface": false }
+}
```
Comment (Fahad-Mahmood) [inline tslint.json]: this is not needed

File: components/Agora/PostCommentModal/PostCommentView.tsx [hunk 3]
```diff
@@ -1,152 +0,0 @@
...
-import styled from 'styled-components';
-import css from '@styled-system/css';
-import axios from 'axios';
-import { useSWRConfig } from 'swr';
-import { Flex, Box, Text } from '@contco/core-ui';
-import { RawViewer } from '@contco/editor';
-import { MsgExecuteContract } from '@terra-money/terra.js';
-import { ModalLarge, Heading3, ButtonRound } from '../../UIComponents';
-import { CLUB_SERVER_ROOT, TEFI_DAGORA_CONTRACT_ADDRESS } from '../../../constants';
-import { simulateSendContractMsg } from '../../../transactions/sendContract';
```
Comment (Fahad-Mahmood) [inline components/Agora/PostCommentModal/PostCommentView.tsx]: this file shouldn't be removed

File: package.json [hunk 4]
```diff
@@ -77,6 +84,24 @@
...
+    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "gh-pages": "^2.2.0",
+    "microbundle-crl": "^0.13.10",
+    "npm-run-all": "^4.1.5",
+    "react-scripts": "^3.4.1",
+    "@babel/core": "^7.11.1",
+    "@types/react-dom": "^16.9.7",
+    "babel-loader": "^8.1.0"
```
Comment (Fahad-Mahmood) [inline package.json]: I don't think these dependencies are required.

**Review by Fahad-Mahmood — COMMENTED** [review]
Looks good, but some fixes are required before we can merge the PR. 🔥 
tefi-app-0.1.1.tgz file should be removed
please add ```yarn lint``` again
packages/editor/dist directory should be removed.
empty interfaces can be removed or [disabled off for the file](https://stackoverflow.com/questions/34764287/turning-off-eslint-rule-for-a-specific-file)

Example Fix for the packages/editor/src/Toolbar/HoveringToolBar.tsx

```
// Code with lint error
export interface HoveringToolBarProps {}

export const HoveringToolBar: React.FC<HoveringToolBarProps> = () => {

// fix

export const HoveringToolBar: React.FC = () => {
```

File: package.json (hunk 4 above)
Comment (abrehmaaan) [reply package.json]: tefi-app-0.1.1.tgz file removed
yarn lint command added again
packages/editor/dist directory is already removed.
empty interfaces are disabled off for the files
dependencies moved to dev dependencies
tslint.json deleted
PostCommentView.tsx file added again
unwanted dependencies removed

**Review by Fahad-Mahmood — APPROVED** [review]

**Pull request merged**